tasmota/xdsp_09_SSD1351.ino

```diff
@@ -59,12 +59,23 @@ void SSD1351_InitDriver() {
     fg_color = SSD1351_WHITE;
     bg_color = SSD1351_BLACK;
 
+
+#ifdef SSD1351_DC
+    int8_t dcpin = SSD1351_DC;
+#else
+    int8_t dcpin = -1;
+#endif
+
+    if (PinUsed(GPIO_SSD1351_DC)) {
+      dcpin = Pin(GPIO_SSD1351_DC);
+    }
+
```

Comment:
There is no need anymore to use this -1 code as function ``Pin()`` now returns ``-1`` if not defined.

So just use ``Pin(GPIO_SSD1351_DC)`` where you now use ``dcpin`` in the code below.

Reply:
ok nice, missed that.